Document software_coexistence for esp32_ble_tracker (#4881)
OttoWinter/esphomedocs: components/esp32_ble_tracker.rst

    you can save power and RF pollution by setting it to ``false``. Defaults to ``true``.
  - **continuous** (*Optional*, boolean): Whether to scan continuously (forever) or to only scan when
    asked to start a scan (with start_scan action). Defaults to ``true``.
  - **software_coexistence** (*Optional*, boolean): When enabled, software coexistence will
    briefly prioritize Bluetooth over Wi-Fi during the initial establishment of BLE connections,

>     which can improve reliability. Has no effect if the ``wifi`` component is not configured. 

Reviewer: ```suggestion
    which can improve reliability. Only available if ``wifi`` component is configured.
```
